datavec/datavec-python/src/main/java/org/datavec/python/PythonContextManager.java

```diff
@@ -157,12 +154,16 @@ public static void deleteContext(String contextName) {
     }
 
     public static void deleteNonMainContexts() {
-        setContext("main");
+        try{
```

Comment:
Describe what a main context is. Also again reference an overarching set of docs for what a context is considered.